.gitignore

```diff
@@ -1,7 +1,11 @@
 .hintrc
 
+.popupformat.html
+
 .stylelintrc.json
 
 package-lock.json 
 
-node_modules
\ No newline at end of file
+node_modules
+
+popupformat.html
```

Comment:
- [ ] Please add `.DS_Store` and `.vscode` in `.gitignore` file so that you can avoid pushing unnecessary file in Github.